jobserver/templates/snapshot_detail.html

```diff
@@ -36,6 +36,13 @@
   </ol>
 </nav>
 
+<h1 class="h3">
+  Outputs for {{ snapshot.workspace.name }}
+  <span class="badge badge-secondary">
+    {% if snapshot.is_draft %}Draft{% endif %}
```

Comment:
Should this entire badge be skipped if it's not draft?

Reply:
Yep, good catch!